app/src/main/java/de/tum/in/tumcampusapp/component/ui/chat/AddChatMemberActivity.java

```diff
@@ -114,20 +122,7 @@ public void onFailure(Call<ChatMember> call, Throwable t) {
                     return;
                 }
 
-                // Get suggestions from backend
-                tumCabeClient.searchChatMember(charSequence.toString(), new Callback<List<ChatMember>>() {
-                    @Override
-                    public void onResponse(Call<List<ChatMember>> call, Response<List<ChatMember>> response) {
-                        searchView.setError(null);
-                        suggestions = response.body();
-                        ((MemberSuggestionsListAdapter)searchView.getAdapter()).updateSuggestions(suggestions);
-                    }
-
-                    @Override
-                    public void onFailure(Call<List<ChatMember>> call, Throwable t) {
-                        onError();
-                    }
-                });
+                delayHandler.postDelayed(suggestionRunnable, DELAY);
```

Comment:
are you sure, this is not executing multiple requests, just delayed? 

Maybe we could use Rxbindings here: https://stackoverflow.com/questions/12142021/how-can-i-do-something-0-5-second-after-text-changed-in-my-edittext/46254925#46254925

Reply:
Ah, nevermind, you are removing the old callback above.